smbc-digital/iag-webapp
# feat(cookie_banner): fix position on mobile/tablet

Moved the cookie banner to the top of the page on mobile and tablet as stated in [Jira D-6882](https://stockportbi.atlassian.net/browse/DIGITAL-6882)

-) Created new "smbc-cookie-banner" style in Old Style Guide, which reverts the cookie banner state to default values, ie top, square border, full width div.
-) Healthy Stockport CookieBanner.cshtml : added new "smbc-cookie-banner" style
-) Stockport Gov CookieBanner.cshtml : added new "smbc-cookie-banner" style
-) Added new styleguide-{sg/hs/ts}.min.css files

To test:

-) Pull the branch and launch
-) Delete "wpcc: dismissed" cookie if you need to ( have already dismissed the cookie )
-) Check the Cookie Banner appears as normal on desktop view, but is a top banner on mobile and tablet
